x-pack/legacy/plugins/uptime/server/graphql/pings/resolvers.ts

```diff
@@ -34,23 +34,23 @@ export const createPingsResolvers: CreateUMGraphQLResolvers = (
 } => ({
   Query: {
     async allPings(
-      resolver,
+      _resolver,
       { monitorId, sort, size, status, dateRangeStart, dateRangeEnd, location },
-      { req }
+      { callAsCurrentUser }
     ): Promise<PingResults> {
-      return await libs.pings.getAll(
-        req,
+      return await libs.pings.getAll({
+        callES: callAsCurrentUser,
         dateRangeStart,
         dateRangeEnd,
         monitorId,
         status,
         sort,
         size,
-        location
-      );
+        location,
+      });
     },
-    async getDocCount(resolver, args, { req }): Promise<DocCount> {
-      return libs.pings.getDocCount(req);
+    async getDocCount(_resolver, _args, { callAsCurrentUser }): Promise<DocCount> {
```

Comment:
Same question about why the underscore on `_args`.

Reply:
https://github.com/elastic/kibana/pull/51403#discussion_r356122938

Reply:
https://github.com/elastic/kibana/pull/51403#discussion_r355708788